ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/precompiles/BLAKE2BFPrecompileContract.java

```diff
@@ -50,10 +50,10 @@ public Gas gasRequirement(final BytesValue input) {
   @Override
   public BytesValue compute(final BytesValue input, final MessageFrame messageFrame) {
     if (input.size() != MESSAGE_LENGTH_BYTES) {
-      return BytesValue.EMPTY;
+      return null;
```

Comment:
(Discussion) What about creating an explicit error and throw a Runtime exception ? What would be the resulting error when you return null ? Is it something meaningful ?

Reply:
Ok sounds good.